cfme/tests/containers/test_overview_data_integrity.py

```diff
@@ -27,42 +25,37 @@
 DataSet = namedtuple('DataSet', ['object', 'name'])
 
 DATA_SETS = [
-    DataSet(Container, 'Containers'),
-    DataSet(NodeCollection, 'Nodes'),
+    DataSet(Node, 'Nodes'),
     DataSet(Project, 'Projects'),
     DataSet(Pod, 'Pods'),
     DataSet(Service, 'Services'),
     DataSet(Route, 'Routes'),
-    DataSet(ImageRegistryAll, 'Registries'),
     DataSet(ContainersProvider, 'Providers')
+    # TDOD: Add Image, Container, ImageRegistry
 ]
 
 
-def get_api_object_counts(providers):
+def get_api_object_counts():
     out = {
         ContainersProvider: 0,
-        Container: 0,
-        NodeCollection: 0,
+        Node: 0,
         Pod: 0,
         Service: 0,
         Project: 0,
-        Route: 0,
-        ImageRegistryAll: 0
+        Route: 0
     }
-    for provider in providers:
+    for provider in list_providers_by_class(ContainersProvider):
```

Comment:
I would like to do this for all the providers since the status box presents objects count for all the provider, but, not all the providers in yaml, I need all the -added- providers, so this a good point and I'll modify accordingly.